templates/decl.j2

```diff
@@ -54,5 +54,8 @@
     </details>
 {% endif %}
 
+{% if decl.sorried %}
+    <p> class="sorry_msg"><strong>Note:</strong> this declaration is incomplete and uses <code>sorry</code>.
```

Comment:
```suggestion
    <p> class="sorry_msg"><strong>Note:</strong> this declaration is incomplete and uses <code>sorry</code>. </p>
```
It doesn't matter if it's the last element before the `</div>`, but maybe somebody else will add another if below this.

Reply:
This was just clumsy, I wouldn't knowingly skip a `<\p>`!

Reply:
It's perfectly valid HTML5 though, the closing tag is implicit in some cases (see "tag omission" in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p).